### DISCUSSION TITLE
Full Thread home coverage and DNS resolve of the Home Assistant server

### DISCUSSION THREAD

**Discussion by fgervais** [openthread/openthread#8722] [Show and tell]
Last summer I deployed 2 instances of `ot-br-posix` border router to get full Thread home coverage but the network was pretty unstable because the 2 BRs would oscillate between 1 and 2 partitions.

I brought this up here and was told that switching to using TREL would solve that issue. (https://github.com/openthread/openthread/discussions/7862).

I did and I'm here to report that it works really well. I keep checking the MLE counter and there is no oscillations anymore.

In case it helps others, here is the topology:
![thread_topology_1 drawio](https://user-images.githubusercontent.com/5411996/216855601-48426890-2b64-461e-9df1-483302ca00a5.png)

The Thread SEDs can also resolve the MQTT server address through DNS so I don't have to hardcode the server address anymore. Nothing special here but since all that technology is still quite new, I'm glad to report that it works.

**Comment by abtink** [comment]
Thanks you @fgervais. Happy to hear this. :)

**Reply by jwhui** [reply]
+1 thanks for testing and reporting back, @fgervais . This is exactly the scenario we wanted to address when developing TREL!
